src/neo/SmartContract/InteropService.Encode.cs

```diff
@@ -0,0 +1,41 @@
+using Neo.Cryptography;
+using Neo.IO;
+using Neo.VM;
+using Neo.Wallets;
+using System;
+using System.Linq;
+
+namespace Neo.SmartContract
+{
+    partial class InteropService
+    {
+        public static class Encode
+        {
+            /// <summary>
+            /// Convert public key to corresponding wallet account address
+            /// </summary>
+            public static readonly InteropDescriptor PubKeyToAddress = Register("Neo.Encode.PubKeyToAddress", Encode_PubKeyToAddress, 0_00010000, TriggerType.All, CallFlags.None);
+
+            /// <summary>
+            /// Calculate corresponding account scripthash for given public key
+            /// </summary>
+            public static readonly InteropDescriptor CreateStandardAccount = Register("System.Contract.CreateStandardAccount", Encode_CreateStandardAccount, 0_00010000, TriggerType.All, CallFlags.None);
```

Comment:
This should be moved to "InteropService.Contract.cs".

Reply:
If we apply this https://github.com/neo-project/neo/issues/1474, which remove base58 address, will it be better to rename it `Neo.Wallet.PublicKeyToAddress`?